doc/source/getting_started.rst

```diff
@@ -1,116 +1,82 @@
-.. _getting-started:
+.. _getting_started:
```

Comment:
This "Getting started" section seems to be pretty much useless. The package-overview is a repetition of the index page as well as the user-side-overview page.

And the section on "Standard Use Cases" will probably be overlooked by many casual users.

So I suggest to revert this page to a simple "Installation" page, and move the overview of standard use cases to the User Guide overview.

Reply:
Renamed the section and integrated the "Standard Use Cases" into the user guide.